### PR TITLE
register and login layout

### PR DESCRIPTION
**Pull request by VivianLi98**
Layout for Login/Register flow. 

Screen-Recording-2022-04-28-at-8

### CODE REVIEW

File: app/src/main/java/com/example/voyagerx/data/LoginRepository.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * Class that requests authentication and user information from the remote data source and
+ * maintains an in-memory cache of login status and user credentials information.
+ */
+
+class LoginRepository(val dataSource: LoginDataSource) {
+
+    // in-memory cache of the loggedInUser object
+    var user: LoggedInUser? = null
+        private set
```
Comment (Celinarabe) [inline app/src/main/java/com/example/voyagerx/data/LoginRepository.kt:14]: How are we getting the current logged in user?

Comment (VivianLi98) [reply app/src/main/java/com/example/voyagerx/data/LoginRepository.kt:14]: That's going to be saved in sharedpreferences. I haven't implemented it yet but will soon.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -10,6 +11,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.VoyagerX">
+        <activity
+            android:name=".LoginActivity"
+            android:exported="true">
```
Comment (joelc1225) [inline app/src/main/AndroidManifest.xml:16]: what does this "exported" attribute do?

File: app/src/main/java/com/example/voyagerx/LoginActivity.kt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        val intendedView = intent.getStringExtra("view")
+        changeView(intendedView?:"login")
+
+    }
+
+    private fun changeView(intendedView: String){
+        val switchLoginRegisterButton = binding.switchLoginRegisterButton
+        val title = binding.loginFlowTitle
+
+        when(intendedView){
```
Comment (joelc1225) [inline app/src/main/java/com/example/voyagerx/LoginActivity.kt:28]: Nice use of `when` statements

**Review by joelc1225 — APPROVED** [review]

**Review by Celinarabe — APPROVED** [review]

**Pull request merged**